content/links/2021-03-29-promcon-online-2021.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "PromCon Online 2021"
+date: 2021-03-29T09:10:21Z
+github_username: tormath1
+twitter_username: tormath1
+link: https://promcon.io/2021-online/schedule/
+hashtags: "#prometheus,#monitoring"
+---
+
+PromCon's schedule is available ! The 3rd of May and online. Which talks do you want to attend ? :)
```

Comment:
What do you think about mentioning `@PromConIO` (their Twitter account)? that could help to share the info with them.

Reply:
Great idea ! Updated issue body; content should follow. :)